packages/ui-widget/README.md

```
@@ -306,10 +356,12 @@ The widget includes a demo server at `demo-server/server.ts` that provides:
 - **Server-Driven UI**: Demonstrates how the widget integrates with a backend
 
 When you run `pnpm dev`, the demo is available at:
+
 - Path-based: http://localhost:3456/u2/login/identifier
```

markdownlint-cli2 (0.20.0)

[warning] Bare URL used

(MD034, no-bare-urls)